llvm/test/CodeGen/AMDGPU/lower-module-lds-zero-size-arr.ll

```diff
@@ -1,14 +1,18 @@
-; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals --version 3
```

Comment:
Why regress the version? Just use --version 5?

Reply:
Bumped to version 5.